.travis.yml

```diff
@@ -9,8 +9,6 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-install:
-  - ./gradlew assemble
```

Comment:
@ashelkovnykov Sorry to ping you on this again, just out of curiosity, what were the following two lines used for in travis, did travis rely on them to compile Photon? And why did we remove them in this PR?
```
install:
  - ./gradlew assemble
```